collector/container.go

```diff
@@ -280,3 +279,15 @@ func (c *ContainerMetricsCollector) collect(ch chan<- prometheus.Metric) (*prome
 
 	return nil, nil
 }
+
+func getContainerIdWithPrefix(containerDetails hcsshim.ContainerProperties) string {
+	switch containerDetails.Owner {
+	case "docker":
+		return "docker://" + containerDetails.ID
+	case "containerd-shim-runhcs-v1.exe":
+		return "containerd://" + containerDetails.ID
+	default:
+		// If not set fall back prior behavior
+		return "docker://" + containerDetails.ID
+	}
```

Comment:
Minor nitpick, but should the `"docker"` case exist if it's already covered by the `default` case?

Reply:
I was going for being overly explicit in the owners,  Happy to go either way.

Reply:
Lets omit the `"docker"` case for now, we can add it in the future if required.